files/zabbix-agent.te

```diff
@@ -1,11 +1,10 @@
-module zabbix-agent 1.0;
+policy_module(zabbix-agent,1.0)
 
 require {
 	type zabbix_agent_t;
-	class process setrlimit;
-	class unix_dgram_socket create;
 }
 
 #============= zabbix_agent_t ==============
 allow zabbix_agent_t self:process setrlimit;
 allow zabbix_agent_t self:unix_dgram_socket create;
+logging_send_syslog_msg(zabbix_agent_t)
```

Comment:
this should probably configureable. Zabbix doesn't write to syslog in every usecase.